src/main/java/com/bullhornsdk/data/model/entity/core/standard/JobOrder.java

```diff
@@ -1487,4 +1487,12 @@ public String toString() {
 				", customObject10s=" + customObject10s +
 				'}';
 	}
+
+	@Override
+	public JobOrder instantiateForInsert() {
+		JobOrder entity = new JobOrder();
+		entity.setEmploymentType("W2");
```

Comment:
W2 is usually Candidate.employeeType!  Easy mistake, just change this to be 'Contract' or 'Permanent' or something.  Only reason I care is most client's systems won't work with that value in that field.